### DISCUSSION TITLE
audio mirroring

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#493] [Q&A]
would audio mirroring be possible with the AoA protocol os something similar? maybe using the website as a usb audio peripheral?

**Comment by yume-chan** [comment]
AOA audio mode turns the Android phone into a microphone, it's not easy to record from it on Web. Also, last time I checked, on Android 10 and newer, the driver is completely broken. Maybe it will work on Android 5 to 8, don't have devices to test.

On Android 11 and newer I have found a method for Scrcpy to record system audio, see https://github.com/Genymobile/scrcpy/issues/14#issuecomment-1406437788 and https://github.com/Genymobile/scrcpy/pull/3703, it should be available in next version.

**Reply by CorruptionVR** [reply]
> it should be available in next version.

On Scrcpy or tango?

**Reply by yume-chan** [reply]
First in Scrcpy (2.0 is shipped with it), then in Tango (basic audio playback has been supported in #495, but still has synchronization issue)

**Reply by CorruptionVR** [reply]
oh, ok. thanks for the clarification.
